### PR TITLE
Add inclusiveness analyzer to CI

### PR DESCRIPTION
**Pull request by fungairino**
## What does this PR do?

- Catch non-inclusive terminology use at PR time.

## Discussion

What settings to adjust? We probably want to use `excludeUnchangedFiles` for now
https://github.com/microsoft/InclusivenessAnalyzer

## Checklist

- [ ] Add tests and/or storybook stories
- [ ] Designate a primary reviewer @mnholtz

### CODE REVIEW

**Comment by fungairino** [comment]
I'll add this to the app as well.

**Comment by twschiller** [comment]
> We probably want to use excludeUnchangedFiles for now

👍

**Review by twschiller — APPROVED** [review]

File: eslint-local-rules/noCrossBoundaryImports.test.js [hunk 1]
```diff
@@ -84,7 +84,7 @@ ruleTester.run("noCrossBoundaryImports", noCrossBoundaryImports, {
       options,
     },
     {
-      // Cross-boundary whitelisted import
+      // Cross-boundary allowed import
```
Comment (twschiller) [inline eslint-local-rules/noCrossBoundaryImports.test.js:87]: Allowlisted

File: src/utils/inference/selectorInferenceUtils.ts [hunk 2]
```diff
@@ -22,7 +22,7 @@
 
 /**
  * Generates a regex to test attribute selectors generated by `css-selector-generator`,
- * to be used in the whitelist/blacklist arrays. The regex will match any selector with or
+ * to be used in the allow/block list arrays. The regex will match any selector with or
```
Comment (twschiller) [inline src/utils/inference/selectorInferenceUtils.ts:25]: Allow/deny-list

File: eslint-local-rules/noCrossBoundaryImports.js [hunk 3]
```diff
@@ -42,7 +42,7 @@ function validateNode({ node, context, boundaries, allowedGlobs }) {
 
   const isAllowedFile = multimatch(importPath, allowedGlobs).length > 0;
   if (isAllowedFile) {
-    // Ignore whitelisted globs like the messenger
+    // Ignore allowed globs like the messenger
```
Comment (twschiller) [inline eslint-local-rules/noCrossBoundaryImports.js:45]: Allowlisted

File: eslint-local-rules/noCrossBoundaryImports.js [hunk 4]
```diff
@@ -42,7 +42,7 @@ function validateNode({ node, context, boundaries, allowedGlobs }) {
 
   const isAllowedFile = multimatch(importPath, allowedGlobs).length > 0;
   if (isAllowedFile) {
-    // Ignore whitelisted globs like the messenger
+    // Ignore allowlisted globs like the messenger
```
Comment (fregante) [inline eslint-local-rules/noCrossBoundaryImports.js:45]: Allowlisted is not a word, let's use "allowed"

Comment (twschiller) [reply eslint-local-rules/noCrossBoundaryImports.js:45]: I prefer allowlisted. It's referring to the allowlist: https://en.wiktionary.org/wiki/allowlisted.

**Pull request merged**